entrypoint_debian.sh

```diff
@@ -46,20 +46,57 @@ fi
 # Copy initial files to host volume
 case ${OPENHAB_VERSION} in
   1.8.3)
-      if [ -z "$(ls -A "${APPDIR}/configurations")" ]; then
+      if [ -z a"$(ls -A "${APPDIR}/configurations")" ]; then
```

Comment:
Why add the 'a' option here? It's not added for the alpine image and it doesn't seem to solve warnings when the dir does not exist.

Reply:
I'm not sure where those a's came from. I did not intentionally ad them. I only made the changes after the comment about the updating. I certainly did not make changes to the 1.8.3 section of the script.